src/Cards/ProductCard.css

```diff
@@ -0,0 +1,102 @@
+a {
+  text-decoration: none;
+  color: inherit;
+}
+.card {
+  color: inherit;
+  background: inherit;
+}
+.product-details {
+  display: flex;
+  flex-direction: column;
+  max-width: 350px;
+  margin: 0.5rem;
+  justify-content: space-between;
+  min-height: 450px;
+  position: relative;
+  text-align: left;
+  cursor: pointer;
+}
+.product-name-and-desc {
+  margin: 0.5rem 0.5rem;
+  flex-grow: 3;
+}
+
+.product-image {
+  height: 300px;
+  width: 100%;
+}
+.red-text {
+  color: red;
+}
+
+.price-details {
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+}
+.price-details > strong,
+strike,
+span {
+  margin-right: 0.35rem;
+}
+.price-details > strike {
+  text-decoration: line-through;
+}
+.float-new {
+  height: 20px;
+  width: 40px;
+  background: pink;
+  position: absolute;
+  top: 20px;
+  padding: 0.3rem 0 0.1rem 0.4rem;
+  color: black;
+}
+.cross {
+  position: absolute;
+  right: 0.5rem;
+  top: 0.5rem;
+  cursor: pointer;
+}
+.product-buttons {
+  display: flex;
+  align-items: center;
+}
+.card button {
+  flex-grow: 4;
+  background: transparent;
+  border: 1px solid grey;
+  margin: 0.25rem 0.25rem;
```

Comment:
`margin: 0.25rem;` will work on all sides (top, right, bottom, left) . You can try to have the shorthand notation here.

Reply:
Done